notebook/static/notebook/js/menubar.js

```diff
@@ -232,15 +232,7 @@ define([
         });
 
         this.element.find('#kill_and_exit').click(function () {
-            var close_window = function () {
-                /**
-                 * allow closing of new tabs in Chromium, impossible in FF
-                 */
-                window.open('', '_self', '');
-                window.close();
-            };
-            // finish with close on success or failure
-            that.notebook.session.delete(close_window, close_window);
+            that.notebook.close_and_halt();
```

Comment:
Now that there's an action for this, it should be possible to remove the handler function, and connect it via `id_actions_dict` below.

Reply:
Ok Thanks! Would it be helpful to change the id from 'kill_and_exit' to 'close_and_halt' in the html for consistency? or vice versa?